From e3b77467de4a6e595a0f6de1ee29eb5c217d36b7 Mon Sep 17 00:00:00 2001
From: Frank Wang <frank.wang@rock-chips.com>
Date: Wed, 29 Mar 2017 18:03:07 +0800
Subject: [PATCH] Revert "usb: ehci-platform: support no relinquishing port
 quirk"

This reverts commit 0fd1853df055.

We found this commit just work well for HS devices, however,
ehci_hub_control will set port owner to companion according line
status bits when FS/LS device is plugged in, so revert this one
and introduce a new workaround.

Change-Id: Ifa856620672191c845abc53a76370cd5bf4097dc
Signed-off-by: Frank Wang <frank.wang@rock-chips.com>
---
 drivers/usb/host/ehci-platform.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/usb/host/ehci-platform.c b/drivers/usb/host/ehci-platform.c
index 77bca46edb86..6f69798e3452 100644
--- a/drivers/usb/host/ehci-platform.c
+++ b/drivers/usb/host/ehci-platform.c
@@ -202,10 +202,6 @@ static int ehci_platform_probe(struct platform_device *dev)
 					  "has-transaction-translator"))
 			hcd->has_tt = 1;
 
-		if (of_property_read_bool(dev->dev.of_node,
-					  "no-relinquish-port"))
-			ehci_platform_hc_driver.relinquish_port = NULL;
-
 		priv->num_phys = of_count_phandle_with_args(dev->dev.of_node,
 				"phys", "#phy-cells");
 
-- 
2.35.3

